Manifold/Expression.swift

```diff
@@ -309,7 +314,7 @@ extension Expression where Recur: FixpointType {
 extension Expression where Recur: FixpointType, Recur: Equatable {
 	// MARK: Typechecking
 
-	public func typecheck(environment: [Name: Expression] = [:]) -> Either<Error, Expression> {
+	public func typecheck(context: [Name: Expression] = [:]) -> Either<Error, Expression> {
```

Comment:
Tonight I was reminded that an `environment` is populated by terms, while a `context` is populated by types.